### DISCUSSION TITLE
this has to be the best out there, thank you! any chance for a mobile version?

### DISCUSSION THREAD

**Discussion by EagleFPV43** [shaked6540/YoutubePlaylistDownloader#441] [General]
thank you for this program, it is amazing how fast it is, the times of using sketchy websites to download videos and songs one by one painfully, taking a good 30 seconds a song are finally over. is there any chance for a mobile version for android? that would be very handy to be able to use it on the go.

**Comment by shaked6540** [comment]
A mobile version is not planned, sadly the libraries I'm using are not compatible with android/iphone.
